sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/schema.yaml

```diff
@@ -2303,3 +2306,30 @@ fields:
 - name: startup_profile_selection_reason_first
   type: STRING
   mode: NULLABLE
+- name: first_document_id
+  type: STRING
+  mode: NULLABLE
+- name: partner_id
+  type: STRING
+  mode: NULLABLE
+- name: distribution_version
+  type: STRING
+  mode: NULLABLE
+- name: distributor
+  type: STRING
+  mode: NULLABLE
+- name: distributor_channel
+  type: STRING
+  mode: NULLABLE
+- name: env_build_platform_version
```

Comment:
I notice a different naming convention e.g. `environment.partner.partner_id` is named `partner_id`, while `environment.build.xpcom_abi` is named `env_build_xpcom_abi` which might cause confusion to users and also differs from the naming in v2. It makes sense to me to keep the naming without the suffix for consistency e.g. `xpcom_abi` instead of `env_build_xpcom_abi`, wdyt?

Reply:
Yeah, some prefix and some don't e.g. partner_id, is_wow64

Reply:
Those are from `environment.partner` and `environment.system` not `environment.build` so doesn't seem like there was a previous convention

Reply:
Unfortunately this leads to an unnecessary complication of needing to update downstream queries when cascading the changes due to different naming in v1 and v2. It'd be so much better if we align the naming between them in this PR rather than updating the schema or expanding the queries later, see e.g. [platfform_version](https://github.com/mozilla/bigquery-etl/blob/b7f199fe643d89bee05ae006e65d5b10e90b0c6a/sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/schema.yaml#L34C25-L34C25)

Reply:
I don't follow. The schemas for `clients_first_seen_v2` and `clients_first_seen_v1` are already very different. Keeping the convention makes sense for the same reason we wouldn't change the upstream `env_build_arch`, `default_search_engine_data_load_path`, or `geo_subdivision1` to match the downstream column names.